packages/types/src/sdk/datasources.ts

```diff
@@ -1,5 +1,7 @@
 import { Table } from "../documents"
 
+export const PASSWORD_REPLACEMENT = "--secret-value--"
```

Comment:
I think we should not show this in the UI when a user adds a secret value - it should be pretty hidden from the user.

Reply:
Sure, do you think it would be better to just not return the property so that it always appears as an empty field in the UI?